src/node/waku_node.py

```diff
@@ -188,17 +188,34 @@ def unpause(self):
 
     def ensure_ready(self, timeout_duration=10):
         @retry(stop=stop_after_delay(timeout_duration), wait=wait_fixed(0.1), reraise=True)
+        def check_healthy(node=self):
+            self.health_response = node.health()
+            if self.health_response == b"Node is healthy":
+                logger.info("Node is healthy !!")
+                return
+            else:
```

Comment:
Styling: Given the `if` arm contains a `return`, I wouldn't use an `else`. But this is a styling preference, up to you :)

Reply:
You are right, I just have only one "else". Fixed at https://github.com/waku-org/waku-interop-tests/pull/35/commits/c61aef2da37d485ea53c93e453e3c1c5aa139289